### PR TITLE
Solution node_ case-tarnsformer-server

### PR DESCRIPTION
**Pull request by NazariiAlieksieiev**

### CODE REVIEW

File: src/createServer.js [hunk 1]
```diff
@@ -1,3 +1,51 @@
...
+      const serverResponse = {
+        originalCase,
+        targetCase,
+        originalText,
+        convertedText,
+      };
+
+      res.statusCode = 200;
+      res.setHeader('Content-Type', 'application/json');
+      res.end(JSON.stringify(serverResponse));
```
Comment (VitaliyBodnar) [inline src/createServer.js:38]: can be less
```suggestion
      res.status(200).send(serverResponse)
```

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]
Almost done!

File: src/createServer.js [hunk 2]
```diff
@@ -1,3 +1,51 @@
...
+    }
+  });
+
+  server.listen(PORT, () => {
+    console.log(`server is running on http://localhost:${PORT}`);
+  });
+
+  server.close();
```
[the remark this answers is not in the document]
Comment (NazariiAlieksieiev) [reply src/createServer.js:46]: I need it to pass tests

File: src/createServer.js [hunk 3]
```diff
@@ -1,3 +1,51 @@
...
+const http = require('http');
+const convertToCase = require('./convertToCase/convertToCase').convertToCase;
+const validationRequest = require('./validationRequest').validationRequest;
+const PORT = process.env.PORT || 3000;
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    const urlParts = req.url.split('?');
+    const originalText = urlParts[0].substring(1);
+    const queryString = urlParts[1];
```
[the remark this answers is not in the document]
Comment (maxim2310) [reply src/createServer.js:13]: not fixed

File: src/createServer.js (hunk 1 above)
Comment (maxim2310) [reply src/createServer.js:40]: not fixed

File: src/request.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-console */
+const axios = require('axios');
+
+const BASE = 'http://localhost:3000';
+const pathName = '/createServer';
+const search = '?toCase=SNAKE';
+
+const href = BASE + pathName + search;
+
+console.log(href);
```
[the remark this answers is not in the document]
Comment (maxim2310) [reply src/request.js:10]: not fixed

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Please check previous mentors comments and try to fix they

**Review by maxim2310 — APPROVED** [review]
Good work!!!

**Pull request closed without merging**